
int16 int_param
float64 double_param
bool is_show_img

bool is_red
int16  ch1_min_r
int16 ch1_max_r
int16  ch1_min_b
int16 ch1_max_b
int16 ch2_min
int16 ch2_max
int16 ch3_min_r
int16 ch3_max_r
int16 ch3_min_b
int16 ch3_max_b
